src/GraphQL/Execution/DefaultExecutionStrategySelector.cs

```diff
@@ -0,0 +1,80 @@
+using GraphQLParser.AST;
+
+namespace GraphQL.Execution
+{
+    /// <inheritdoc cref="IExecutionStrategySelector"/>
+    public class DefaultExecutionStrategySelector : IExecutionStrategySelector
+    {
+        private readonly ExecutionStrategyRegistration[] _registrations;
+
+        /// <summary>
+        /// Initializes an instance that only returns the default registrations;
+        /// <see cref="ParallelExecutionStrategy"/> for <see cref="OperationType.Query"/> and
+        /// <see cref="SerialExecutionStrategy"/> for <see cref="OperationType.Mutation"/>.
+        /// </summary>
+        public DefaultExecutionStrategySelector()
+            : this(Array.Empty<ExecutionStrategyRegistration>())
+        {
+        }
```

Comment:
Could make this `internal` as it is only used when `DocumentExecuter` is not provided an instance of `IExecutionStrategySelector`.  Could also make this a static instance, but it shouldn't be used with DI at all, so I figured this was better.

Reply:
Per your comments, I will make this internal.

Reply:
Made internal.  Lmk if you want it public or resolve if not.